packages/contracts/src/contracts/tornado-anchor.ts

```diff
@@ -177,10 +177,10 @@ export class TornadoContract {
   async generateZKP(deposit: Deposit, zkpPublicInputs: ZKPTornPublicInputs) {
     const merkleProof = await this.generateMerkleProof(deposit);
     const { pathElements, pathIndex: pathIndices, root } = merkleProof;
-    let circuitData = require('../circuits/withdraw.json');
-    let proving_key = require('../circuits/withdraw_proving_key.bin');
-    proving_key = await fetch(proving_key);
-    proving_key = await proving_key.arrayBuffer();
+    let circuitDataResponse = await fetch('https://ipfs.io/ipfs/QmbX8PzkcU1SQwUis3zDWEKsn8Yjgy2vALNeghVM2uh31B');
```

Comment:
Let's create a constants file and put the hashes there?

Reply:
I put these in files: packages/apps/src/configs/ipfs